DOC: Update gui.md with example on how to display a node in only some views (#7708)
Slicer/Slicer: Docs/developer_guide/script_repository/gui.md

sliceController.barLayout().addWidget(myButton)
```

> ### Display a node in only some views
> This script displays an object that has a DisplayNode in a predefined set of views (here Red+, Green+ and Yellow+)
> ```python
> sliceNodeRed_plus = slicer.app.layoutManager().sliceWidget("Red+").mrmlSliceNode()
> sliceNodeGreen_plus = slicer.app.layoutManager().sliceWidget("Green+").mrmlSliceNode()
> sliceNodeYellow_plus = slicer.app.layoutManager().sliceWidget("Yellow+").mrmlSliceNode()
> 
> my_node.GetDisplayNode().SetViewNodeIDs([sliceNodeRed_plus.GetID(), sliceNodeGreen_plus.GetID(), sliceNodeYellow_plus.GetID()])
> ```

Reviewer: ````suggestion
### Display a node in only some views

Show a displayable node in a predefined set of views. Here, show a markups line only in `Red+`, `Green+` and `Yellow+` views.

```
displayableNode = getNode("L")

viewNodeIDs = [
  slicer.app.layoutManager().sliceWidget(viewName).mrmlSliceNode().GetID()
  for viewName in ["Red+", "Green+", "Yellow+"]
]

displayableNode.GetDisplayNode().SetViewNodeIDs(viewNodeIDs)
````

For reference, here is a related issues:
* https://github.com/Slicer/Slicer/issues/7434

sliceController.barLayout().addWidget(myButton)
```

### Display a node in only some views

Show a displayable node in a predefined set of views. Here, show a markups line only in `Red+`, `Green+` and `Yellow+` views.

```python
displayableNode = getNode("L")

viewNodeIDs = [
  slicer.app.layoutManager().sliceWidget(viewName).mrmlSliceNode().GetID()
  for viewName in ["Red+", "Green+", "Yellow+"]
]

> displayableNode.GetDisplayNode().SetViewNodeIDs(viewNodeIDs)

Reviewer: ````suggestion
displayableNode.GetDisplayNode().SetViewNodeIDs(viewNodeIDs)
```
````
